addons/mail/static/src/discuss/call/common/call_participant_card.js

```diff
@@ -75,6 +75,16 @@ export class CallParticipantCard extends Component {
         return Boolean(this.rtcSession && this.rtcSession.channel?.eq(this.rtc.state.channel));
     }
 
+    get showMissingAudioTrack() {
+        return (
+            this.rtcSession &&
+            this.isOfActiveCall &&
+            !this.rtcSession.eq(this.rtc?.selfSession) &&
+            !this.rtcSession.isSelfMuted &&
+            !this.rtcSession.audioStream
+        );
+    }
```

Comment:
Nitpicking: I would probably rename to `probablyMissingAudioStream/Track`. The "probably" prefix helps understanding this is an heuristic of a technical issue, which may happens (in theory) for reasons!

Again test coverage would justify having these heuristics and UI for handling this case.
If this feels relevant the test ensures this is kept, if we feel it's unnecessary then the test gives some confidence on what sort of scenario the heuristics is covering and we can just scrap the test and related LOCs.